feature/office/src/main/java/com/iteneum/office/presentation/ui/OfficeView.kt

```diff
@@ -35,7 +35,7 @@ fun OfficeView() {
         viewModel.getOfficeInformation()
     }
 
-    val officeData =  viewModel.officeInfo
+    val officeData = viewModel.officeInfo
 
     officeData?.let {
```

Comment:
Remember to make a reformatting code for easy reading.  Use **ctlr + L** to reformat code